tests/lcs/core/graphics/graphics.livecodescript

```diff
@@ -90,4 +90,163 @@ TestAssert "test", not ("4,9" is not within "-5,-10,5,10")
 TestAssert "test", "-6,-11" is not within "-5,-10,5,10"
 TestAssert "test", "5,10" is not within "-5,-10,5,10"
 
-end TestGraphicsIsNotWithin
\ No newline at end of file
+end TestGraphicsIsNotWithin
+
+-- new brush mapping for 101..135
+-- brush < 101 = brush
+-- 101 < brush < 135 = brush - 100
+-- brush > 136 = brush
+
+on TestBrushWithNoIcons
+	TestAssert "set brush with revicons stack not loaded", SetBrushNoIcons() is not 2
+end TestBrushWithNoIcons
+
+on TestInitialPaintConditions
+#	open stack TestGetIDERepositoryPath() & "/Toolset/palettes/revicons.rev"
+	TestAssert "brush is initially 8", the brush is 8
+	TestAssert "spray is initially 34", the spray is 34
+	TestAssert "eraser is initially 2", the eraser is 2
+	
+	TestAssert "the centered is initially false", the centered is false
+	TestAssert "the filled is initially false", the filled is false
+	
+	TestAssert "the grid is initially true", the grid is true
+
+	TestAssert "the gridSize is initially 4", the gridSize is 4
+	TestAssert "the lineSize is initially 0", the lineSize is 0
+
+	TestAssert "the multiple is initially false", the multiple is false
+	TestAssert "the multiSpace is initially 1", the multiSpace is 1
+
+# not sure if this should work. it doesn't in previous builds either
+	TestAssert "the penPattern is initially 136", the penPattern is 136
```

Comment:
Nonetheless, the documentation says, and has always said, that the brushes are between 1 and 35. All I did was bring the remaining errant spray doc in line, since it was quite wrong to begin with. We can start making 136 a brush if you want (and I reiterate that if we remove the mapping then it doesn't matter what we call these things, we can just set the id value and use it as a brush or as a pattern, the way we can do now with other image ids not in the special set.